snap/snapcraft.yaml

```diff
@@ -368,8 +368,8 @@ parts:
       - libzmq3-dev
   support-scheduler:
     source: https://github.com/edgexfoundry/support-scheduler.git
-    # the following commit == 0.5.0 version update +1 (core-domain fix)
-    source-commit: "00707ae"
+    # the following commit == 0.7.0 version update
+    source-commit: "194094a3cf2a0a3a2cc608cec7cd3c4904ac6b5b"
```

Comment:
Is there a tag that corresponds to this commit, or are we still forced to use a commit SHA? Also any reason you switched to using the full SHA vs. a short SHA?